Update docs based on 7.0 cloud provisioner config testing

wadells opened the pull request:
## Description
Minor docs tweaks found while walking through part of the 7.0 test plan.  The individual commit messages have more context for each change on its own.

## Type of change
* Docs
* Bug fix (non-breaking change which fixes an issue)
* This change has a user-facing impact

## Linked tickets and other PRs
All this cleanup was discovered while working on #1826 

## TODOs
- [x] Self-review the change
- [x] Perform manual testing
- [x] Address review feedback

## Testing done
<details><summary><code>make docs; make docs-lint</code></summary>

```
walt@work:~/git/gravity$ make docs-lint
make -C docs lint
make[1]: Entering directory '/home/walt/git/gravity/docs'
docker run --rm=true -u $(id -u):$(id -g) -v "$(pwd)/../":/home -w /home/docs -h docs docs-buildbox:latest make container-lint
cd 4.x && milv -ignore-external
NO ISSUES :-)
cd 5.x && milv -ignore-external
NO ISSUES :-)
cd 6.x && milv -ignore-external
NO ISSUES :-)
cd 7.x && milv -ignore-external
NO ISSUES :-)
make[1]: Leaving directory '/home/walt/git/gravity/docs'
walt@work:~/git/gravity$ make docs
make -C docs
make[1]: Entering directory '/home/walt/git/gravity/docs'
docker run --rm=true -u $(id -u):$(id -g) -v "$(pwd)/../":/home -w /home/docs -h docs docs-buildbox:latest make container-compile-docs
mkdocs build --strict --config-file 4.x.yaml --site-dir ../build/docs/4.x
INFO    -  Cleaning site directory 
INFO    -  Building documentation to directory: /home/build/docs/4.x 
INFO    -  The following pages exist in the docs directory, but are not included in the "nav" configuration:
  - testplan.md
  - workshops.md 
mkdocs build --strict --config-file 5.x.yaml --site-dir ../build/docs/5.x
INFO    -  Cleaning site directory 
INFO    -  Building documentation to directory: /home/build/docs/5.x 
INFO    -  The following pages exist in the docs directory, but are not included in the "nav" configuration:
  - testplan.md
  - workshops.md 
mkdocs build --strict --config-file 6.x.yaml --site-dir ../build/docs/6.x
INFO    -  Cleaning site directory 
INFO    -  Building documentation to directory: /home/build/docs/6.x 
INFO    -  The following pages exist in the docs directory, but are not included in the "nav" configuration:
  - testplan.md
  - workshops.md 
mkdocs build --strict --config-file 7.x.yaml --site-dir ../build/docs/7.x
INFO    -  Cleaning site directory 
INFO    -  Building documentation to directory: /home/build/docs/7.x 
INFO    -  The following pages exist in the docs directory, but are not included in the "nav" configuration:
  - ingress.md
  - testplan.md
  - workshops.md 
cd ../build/docs && rm -f latest && ln -s 7.x latest
make[1]: Leaving directory '/home/walt/git/gravity/docs'
```
</details>

I also tested that the `#managing-an-ongoing-operation` id works correctly via `make run-docs`.  Unfortunately, I don't have a good way to show this (without a video or something) so you'll have to take my word for it.

Lastly,  despite my attempt at improving our `--cloud-provider` documentation, I still have not been able to correctly set up a cluster with 100% working cloud provider in AWS or GCP.  Fortunately, I didn't need working networking to complete the test plan.... so I'm posting what I have, and hopefully the next engineer who runs this test plan can further build on the level of detail.

wadells committed:
Minor tweaks to requirements.

Consistency of GB vs GiB. Clarity about which nodes need an etcd disk.

wadells committed:
Fix a typo in a log message.

wadells committed:
Update resource operation test plan.

Minor tweaks based on my experience walking through this test plan.

wadells committed:
Fix broken url fragment.

https://gravitational.com/gravity/docs/cluster/#managing-operations
is one of the first things new users will see (it shows up during
install).  Best to make sure it points to a real place.

wadells committed:
Refactor `--cloud-provider` install docs.

I've been having difficulty properly provisioning a cloud provider
integrated cluster. This is a slight rewording of the docs to make
them more consistent and link out to useful resources.

bernardjkim commented on docs/7.x/testplan.md:
````diff
@@ -421,26 +427,28 @@ kind: ClusterConfiguration
       node-tags=test-cluster
 ```
 
+Note: `cloudProvider:`'s value depends on which provider the cluster is provisioned on.
````
nit: colon after cloudProvider

wadells replied:
Fixed in 88b1c10.

a-palchikov commented on docs/7.x/requirements.md:
```diff
@@ -134,10 +134,10 @@ These ports are used for Cluster operation and should be open between cluster no
+    * all - Any node which is a member of the cluster
+    * ext - Any source outside the cluster
+    * localhost - The port is only used within the host where the request started
+    * controllers - Nodes which are designated "controller" (aka "master") role
```
same nit as in https://github.com/gravitational/gravity/pull/1876

wadells replied:
> nit: Maybe use control plane instead of controller when describing a controller node?

We don't use control plane consistently elsewhere in the docs, so I'm not sure using it here will be any clearer than the admittedly muddy `controllers`.

```
walt@work:~/git/gravity$ rg 'control plane' docs/7.x
docs/7.x/hub.md
3:The Gravity Hub is a multi-cluster control plane available in the Enterprise

docs/7.x/cluster.md
732:Clusters with complex software deployed on them require separation of workloads between the control plane and the application components to enable a seamless upgrade experience.
1428:### Dedicated control plane nodes
```

`master` seems to be the most commonly used terminology.  Does that work for you?

wadells replied:
Aside from this, the intent touching this line is fixing a blatantly broken indentation level:

![image](https://user-images.githubusercontent.com/187314/87588124-f8844200-c697-11ea-9a86-d6101dfac1eb.png)

I'm not sure I want to take on this scope creep, as changing this will require editing all of the table above.

wadells replied:
I've decided not to pursue the recommended change in this PR.  Too much scope creep.

wadells committed:
Add backwards compatible html id.

Gravity binaries in the wild will link the #managing-an-ongoing-operation
fragment for years to come.  Add a backwards compatibility hack to let
them jump to the right section.
